django_declarative_apis/machinery/__init__.py

```diff
@@ -228,7 +230,9 @@ def get_response(self):  # noqa: C901
 
             resource = self.bound_endpoint.resource
 
-            if hasattr(resource, "is_dirty"):
+            if isinstance(resource, django.db.models.Model) and hasattr(
```

Comment:
The idea is that if it's not a Django Model we might not want to update it so often?  Updates for those objects will have to be handled outside of DDA?

Reply:
Ah- in `finalize`

Reply:
yeah, this auto-save functionality is pretty tightly coupled to the resources being Django models.  This PR allows using non-django models as resources, but the cost is that the endpoint implementation must handle persistence on its own